doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md

```diff
@@ -0,0 +1,50 @@
+#<a name="top"></a>Grouping Rules
+The file `grouping_rules.conf` can be instantiated from a template given in the Cygnus repository, `conf/grouping_rules.conf.template`.
```

Comment:
Start with a line similar to the name mappings one:

Name mappings is an advanced global feature of Cygnus. It is global because it is available for all NGSI sinks.